src/panels/config/backup/components/config/ha-backup-config-retention.ts

```diff
@@ -54,16 +55,21 @@ class HaBackupConfigRetention extends LitElement {
 
   @state() private _value = 3;
 
+  @query("#value") private _customValueField?: HaMdTextfield;
+
+  @query("#type") private _customTypeField?: HaMdSelect;
+
+  private _configLoaded = false;
+
   private presetOptions = [
     RetentionPreset.COPIES_3,
     RetentionPreset.FOREVER,
     RetentionPreset.CUSTOM,
   ];
 
-  public willUpdate(properties: PropertyValues) {
-    super.willUpdate(properties);
-
-    if (!this.hasUpdated) {
+  public willUpdate() {
+    if (!this._configLoaded && this.retention !== undefined) {
```

Comment:
I think this could also be solved by checking `retention === undefined`, if it loaded and unset it should be `null`, so is the `_configLoaded` really needed?

Reply:
Mhh I don't get it. I want it when `retention !== undefined` but this would happen on every change, I just want it once, because of that I added the helper var.